vi/news/_posts/2016-12-25-ruby-2-4-0-released.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: news_post
+title: "Ruby 2.4.0 Released"
+author: "naruse"
+translator:
+date: 2016-12-25 00:00:00 +0000
+lang: vi
+---
+
+Chúng tôi rất hân hạnh được thông báo về việc phát hành Ruby phiên bản 2.4.0.
+
+Ruby 2.4.0 là phiên bản stable đầu tiên của Ruby 2.4.0 gồm nhiều tín năng mới như:
+
+## [Giới thiệu cải tiến hash table (bởi Vladimir Makarov)](https://bugs.ruby-lang.org/issues/12142)
+
+Cải thiện cấu trúc bên trong của hash table (st_table)
+Improve the internal structure of hash table (st_table) bằng việc giới thiệu
+open addressing và một mảng bao hàm có thứ tự.
+Cải thiện này đã được rất nhiều người chấp nhận, đặc biệt là Yura Sokolov.
+
+## Binding#irb: Bắt đầu một REPL session tương tự như `binding.pry`
+
+Thông thường bạn có thể dùng `p` để xem giá trị của các biến trong quá trình debug.
+Với [pry](https://github.com/pry/pry) bạn có thể dùng `binding.pry` để chạy REPL và
+debug ở bất kỳ dòng code nào.
+[r56624](https://github.com/ruby/ruby/commit/493e48897421d176a8faf0f0820323d79ecdf94a) giới thiệu `binding.irb` hoạt động tương tự IRB.
```

Comment:
This might cause confusion `hoạt động tương tự IRB`
My suggestion is: `có tính năng tương tự trong IRB`